scripts/init-submodules-no-riscv-tools-nolog.sh

```diff
@@ -4,6 +4,14 @@
 set -e
 set -o pipefail
 
+# Check that git version is at least 1.7.8
+MYGIT=$(git --version | awk '{print $3;}')
```

Comment:
If somehow `MYGIT` isn't a version string but begins with a non-numeric character, then it would pass the check.  This probably isn't worth considering because if the `git --version` output ever changed, then it would imply a sufficiently new git.

Although overkill, we could make the version parsing more robust like so:
```suggestion
MYGIT=$(git --version)
MYGIT=${MYGIT#'git version '} # Strip prefix
case ${MYGIT} in
[1-9]*) ;;
*) echo 'warning: unknown git version' ;;
esac
```

Reply:
Cool, thanks for the improvement- applied (and tested locally just to be diligent).